apollo-plugin/apollo-plugin-log4j2/README.md

```diff
@@ -0,0 +1,38 @@
+# Apollo Log4j2
+
+This module could let you integrate log4j2 with apollo easily.
+You could create a namespace `log42.xml`, then log4j2 will load the configuration from it.
+
+## How to use it?
+
+There are several steps that need to do:
+
+1. Add log4j2 dependency to your classpath
+   ```xml
+   <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-api</artifactId>
+   </dependency>
+   <dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-core</artifactId>
+   </dependency>
+   ```
+2. Add `apollo-plugin-log4j2` dependency to your classpath
+   ```xml
+   <dependency>
+       <groupId>com.ctrip.framework.apollo</groupId>
+       <artifactId>apollo-plugin-log4j2</artifactId>
+   </dependency>
+   ```
+3. Create a new namespace `log4j2.xml` (namespace format must be XML) in your apollo application
+4. Add system properties `apollo.log4j2.enabled` when you run java application
+   ```bash
+   -Dapollo.log4j2.enabled=true
```

Comment:
Is it better to enable it by default? As the user already imports this maven dependency, I suppose the intention is to enable this feature?

Reply:
Yes, enabling it by default is better now.

Reply:
Oops! I know I disabled it by default because it could load the `log4j2.xml` from the classpath when we test code in the local environment. It prevents loading the `log4j2.xml` from Apollo which often does not satisfy the personal environment, such as the UNIX log path is not satisfy windows users.

I think we could support reading config from env `APOLLO_LOG4j2_ENABLED`, then users could enable it by default. What do you think? @nobodyiam

Reply:
I think it's ok.